src/seed.js

```diff
@@ -0,0 +1,35 @@
+// import { envConfig } from './configs/env.js';
+import mongoose from 'mongoose';
+import User from './models/UserModel.js';
+import databaseConnect from './dbConnection.js';
+
+databaseConnect()
+  .then(async () => {
+    await User.collection.drop();
+
+    // eslint-disable-next-line no-console
+    console.log('Creating seed data');
+
+    let adminUser = await User.create({
+      email: 'AdminUser@email.com',
+      password: 'AdminUser1',
+      role: 'admin'
+    });
+
+    // eslint-disable-next-line no-console
+    console.log(adminUser);
+
+    let user1 = await User.create({
+      email: 'user1@email.com',
+      password: 'User1',
+      role: 'user'
+    });
+
+    // eslint-disable-next-line no-console
+    console.log(user1);
+  })
+  .then(async () => {
+    await mongoose.connection.close();
+    // eslint-disable-next-line no-console
+    console.log('Database disconnected!');
+  });
```

Comment:
Are we lacks error handling here? If any operation (like dropping the collection or creating users) fails, the script might exit unexpectedly.

Reply:
I have not done error handling for any code yet, will make this all work first and create an error handling middleware or some kinds of common functions for error handling. That's the next tasks. For now, I focus on make the seeding works first and database are correctly designed.

Reply:
Okay, I got it. Thank you